packages/expo-module-template/$package.json

```diff
@@ -18,7 +23,8 @@
   "homepage": "<%- repo %>#readme",
   "dependencies": {},
   "devDependencies": {
-    "expo-module-scripts": "^2.0.0"
+    "expo-module-scripts": "^2.0.0",
+    "expo-modules-core": "^0.6.5"
```

Comment:
- Hmm, `expo-modules-core` is a dependency of `expo` which is a peer dependency of this package 🤔
- `expo-modules-core` is now 0.7.0

Reply:
good point. would a better fix be for one of the expo-module scripts to install peer dependencies?

Reply:
Honestly, I'm not sure. Let's wait for @tsapeta review

Reply:
I agree that not having `expo` or `expo-modules-core` in `devDependencies` may result in some issues when the module is not created within the workspace/monorepo that already has those packages hoisted. However, I'm not really convinced to depend on the specific version of `expo-modules-core` — I would be in favor of depending on the `expo` package instead because it's already a peer dependency and it provides `expo-modules-core` as well. Could you check if that works?